tests/setup_nodes.py

```diff
@@ -650,19 +667,20 @@ async def setup_full_system(
                 node2_port,
                 rpc2_port,
                 b_tools_1,
-                introducer_port,
-                False,
-                10,
-                True,
-                False,  # connect_to_daemon,
+                connect_to_daemon_port=None,
+                introducer_port=introducer_port,
+                simulator=False,
+                send_uncompact_interval=10,
+                sanitize_weight_proof_only=True,
                 db_version=db_version,
             ),
             setup_vdf_client(shared_b_tools, shared_b_tools.config["self_hostname"], vdf2_port),
             setup_timelord(timelord1_port, 1000, timelord1_rpc_port, vdf2_port, True, consensus_constants, b_tools_1),
         ]
 
-        if connect_to_daemon:
+        if connect_to_daemon_port:
             node_iters.append(setup_daemon(btools=b_tools))
+            daemon1 = await node_iters[9].__anext__()
```

Comment:
This moves the daemon to be created first

Reply:
I wouldn't expect the *order* they are created to matter. Assuming they've all been configured with an available listen port already.

Reply:
No, just hoped to speed connection up. Not sure what the poll interval is